.swiftformat

```diff
@@ -0,0 +1,118 @@
+# file options
```

Comment:
are we aligned on using swiftformat?

Reply:
https://github.com/awslabs/aws-crt-swift/pull/70/files

we would need replacement for CI also, like swiftlint does today.